### PR TITLE
Add Logging Sample

### PR DESCRIPTION
**Pull request by dzou**
Adds a basic API logging sample.

### CODE REVIEW

**Review by meltsufin — APPROVED** [review]
So, no extra graal config to make this work? Cool!

**Comment by dzou** [comment]
> So, no extra graal config to make this work? Cool!

Yup that's right, I think this is gonna be the case for most (if not, all) of the libraries that offer only GAPIC generated APIs. For the libraries with handwritten layers this might be different though.

File: google-cloud-graalvm-samples/logging-sample/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+   mvn package -P graal
+   ```
+
+2. Run the application:
+
+   ```
+   ./target/com.example.loggingsampleapplication
+   ```
+
+3. The application will log a message. Navigate to the [Cloud Console Logs Viewer](https://console.cloud.google.com/logs/viewer) to view the newly generated log entry.
```
Comment (ttomsu) [inline google-cloud-graalvm-samples/logging-sample/README.md:27]: >will log a message.

To the console? To the Cloud console? Both? I'd specify the expectation.

I'd also specify what what was logged so someone who is using this with another noisy product could pick it out from their other noise.

File: google-cloud-graalvm-samples/logging-sample/pom.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  </profiles>
+
+  <repositories>
+    <repository>
+      <id>spring-milestone</id>
+      <name>Spring milestone</name>
+      <url>https://repo.spring.io/milestone</url>
+    </repository>
+  </repositories>
+</project>
```
Comment (ttomsu) [inline google-cloud-graalvm-samples/logging-sample/pom.xml:121]: nit: newline

Looks like someone needs a checkstyle ;-)

File: google-cloud-graalvm-samples/logging-sample/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Navigate to this directory in a new terminal.
+
+1. Compile the application using the GraalVM Compiler. This step may take a few minutes.
+
+   ```
+   mvn package -P graal
+   ```
+
+2. Run the application:
```
Comment (ttomsu) [inline google-cloud-graalvm-samples/logging-sample/README.md:21]: It looks like one may need to run `gcloud auth application-default login` to make the authorization work without a service account?

**Review by ttomsu — APPROVED** [review]

**Pull request merged**